detekt/detekt.bzl

```diff
@@ -12,8 +12,9 @@ def _impl(ctx):
     action_arguments = [
         "-Xms16m",
         "-Xmx128m",
-        "-jar",
-        ctx.file._detekt_cli_jar.path,
+        "-cp",
+        ":".join([jar for jar in [ctx.file._detekt_cli_jar.path, ctx.file._detekt_wrapper_jar.path]]),
```

Comment:
This should work (at least in actual Python).

```python
":".join(["a", "b", "c"])
```

Reply:
I'll leave this one to you to get you onto Bazel stuff